tests/unit/pages/embeds/Plio.spec.js

```diff
@@ -318,4 +319,96 @@ describe("Plio.vue", () => {
     expect(checkAndSetPlayerAspectRatio).toHaveBeenCalled();
     expect(wrapper.vm.isAspectRatioChecked).toBeTruthy();
   });
+
+  it("calls updateEvent method for consecutive 'watching' events for a given session", async () => {
+    const plioId = "123";
+
+    // mock setPlayerAspectRatio method
+    jest
+      .spyOn(Plio.methods, "setPlayerAspectRatio")
+      .mockImplementation(() => jest.fn());
+
+    // mock createSession method
+    jest.spyOn(Plio.methods, "createSession").mockImplementation(() => {
+      return new Promise((resolve) => resolve());
+    });
+
+    // mock updateEvent method
+    let updateEvent = jest
+      .spyOn(Plio.methods, "updateEvent")
+      .mockImplementation(() => {
+        return;
+      });
+
+    // mock user service
+    jest
+      .spyOn(UserAPIService, "getUserByAccessToken")
+      .mockImplementation(() => {
+        return new Promise((resolve) => {
+          resolve({ data: dummyUser });
+        });
+      });
+
+    // set user
+    await store.dispatch("auth/setAccessToken", dummyAccessToken);
+
+    mount(Plio, {
+      shallow: true,
+      props: {
+        plioId: plioId,
+      },
+      data() {
+        return {
+          watchingEventDBId: 1,
```

Comment:
comment saying that this will simulate the case when watching event was already created